### PR TITLE
Change commit status icons to SVG

### PR DESCRIPTION
**Pull request by silverwind**
Screen Shot 2022-08-09 at 19 32 24
Screen Shot 2022-08-09 at 19 32 33
Screen Shot 2022-08-09 at 19 32 42
Screen Shot 2022-08-09 at 19 33 03
Screen Shot 2022-08-09 at 19 50 26

### CODE REVIEW

File: web_src/js/features/repo-commit.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ export function initCommitStatuses() {
 
     createTippy(this, {
       trigger: 'click',
-      content: this.nextSibling,
+      content: this.nextElementSibling,
```
Comment (silverwind) [inline web_src/js/features/repo-commit.js]: This is the actual bug fix, `nextSibling` did include HTML text nodes,  `nextElementSibling` doesn't do this.

**Comment by silverwind** [comment]
Bugfix split out into https://github.com/go-gitea/gitea/pull/20737.

**Review by Gusted — APPROVED** [review]

**Comment by silverwind** [comment]
Positioning also changed to `top-start` and `bottom-start` as per https://github.com/go-gitea/gitea/pull/20428#issuecomment-1209747840, so that the popover does not overlap with the commit message when it wraps.

**Comment by silverwind** [comment]
One more notable change, the `error` and `warning` cases now use this icon:

image

Previously, it was

image

I would have liked to use a filled variant, but Octicons does not have it currently.

**Review by delvh — APPROVED** [review]

**Comment by Gusted** [comment]
> I would have liked to use a filled variant, but Octicons does not have it currently.

Create a Gitea variant?

**Comment by silverwind** [comment]
> > I would have liked to use a filled variant, but Octicons does not have it currently.
> 
> Create a Gitea variant?

Did it, but actually I think it looks a bit odd:

image

So I think we can keep the outline variant, or maybe I find something better :)

**Comment by Gusted** [comment]
> Did it, but actually I think it looks a bit odd because we have no fill icons there otherwise:

Uh yeah quite odd, the normal one is more "minimalist" and feels more like a fit.

**Comment by silverwind** [comment]
Added a exclamation icon, in the spirit of the old icon:

image

**Comment by 6543** [comment]
.

**Pull request merged**